datasets/ecthr_cases/ecthr_cases.py

```diff
@@ -0,0 +1,200 @@
+    author = "Chalkidis, Ilias and Fergadiotis, Manos and Tsarapatsanis, Dimitrios and Aletras, Nikolaos and Androutsopoulos, Ion and Malakasiotis, Prodromos",
+    booktitle = "Proceedings of the Annual Conference of the North American Chapter of the Association for Computational Linguistics",
+    year = "2021",
+    address = "Mexico City, Mexico",
+    publisher = "Association for Computational Linguistics"
+}
+"""
+
+_DESCRIPTION = """\
+The ECtHR Cases dataset is designed for experimentation of neural judgment prediction and rationale extraction considering ECtHR cases.
+"""
+
+_HOMEPAGE = "http://archive.org/details/ECtHR-NAACL2021/"
+
+_LICENSE = "CC BY-NC-SA (Creative Commons / Attribution-NonCommercial-ShareAlike)"
+
+_URLs = {
+    "alleged-violation-prediction": "http://archive.org/download/ECtHR-NAACL2021/dataset.zip",
+    "violation-prediction": "http://archive.org/download/ECtHR-NAACL2021/dataset.zip",
+}
+
+ARTICLES = {
+    "2": "Right to life",
+    "3": "Prohibition of torture",
+    "4": "Prohibition of slavery and forced labour",
+    "5": "Right to liberty and security",
+    "6": "Right to a fair trial",
+    "7": "No punishment without law",
+    "8": "Right to respect for private and family life",
+    "9": "Freedom of thought, conscience and religion",
+    "10": "Freedom of expression",
+    "11": "Freedom of assembly and association",
+    "12": "Right to marry",
+    "13": "Right to an effective remedy",
+    "14": "Prohibition of discrimination",
+    "15": "Derogation in time of emergency",
+    "16": "Restrictions on political activity of aliens",
+    "17": "Prohibition of abuse of rights",
+    "18": "Limitation on use of restrictions on rights",
+    "34": "Individual applications",
+    "38": "Examination of the case",
+    "39": "Friendly settlements",
+    "46": "Binding force and execution of judgments",
+    "P1-1": "Protection of property",
+    "P1-2": "Right to education",
+    "P1-3": "Right to free elections",
+    "P3-1": "Right to free elections",
+    "P4-1": "Prohibition of imprisonment for debt",
+    "P4-2": "Freedom of movement",
+    "P4-3": "Prohibition of expulsion of nationals",
+    "P4-4": "Prohibition of collective expulsion of aliens",
+    "P6-1": "Abolition of the death penalty",
+    "P6-2": "Death penalty in time of war",
+    "P6-3": "Prohibition of derogations",
+    "P7-1": "Procedural safeguards relating to expulsion of aliens",
+    "P7-2": "Right of appeal in criminal matters",
+    "P7-3": "Compensation for wrongful conviction",
+    "P7-4": "Right not to be tried or punished twice",
+    "P7-5": "Equality between spouses",
+    "P12-1": "General prohibition of discrimination",
+    "P13-1": "Abolition of the death penalty",
+    "P13-2": "Prohibition of derogations",
+    "P13-3": "Prohibition of reservations",
+}
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class ECtHRCases(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class EcthrCases(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `etchr_cases`